src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -172,6 +172,15 @@ <h3 class="top-container flex-wrap flex-lg-nowrap">
                 <fa-icon *ngIf="submitBusy" [icon]="faSpinner" [spin]="true">&nbsp;</fa-icon>
                 <span jhiTranslate="artemisApp.assessment.button.overrideAssessment">Override Assessment</span>
             </button>
+            <button
+                class="btn m-1 btn-primary"
+                id="useAsExampleSubmission"
+                *ngIf="result?.completionDate && isAtLeastInstructor && (exercise!.type === ExerciseType.MODELING || exercise!.type === ExerciseType.TEXT)"
+                (click)="onUseAsExampleSolutionClicked()"
+            >
+                <fa-icon *ngIf="submitBusy" [icon]="'spinner'" [spin]="true"></fa-icon>
```

Comment:
Please don't forget to think about the new way we handle icons.
Have a look here if it is needed here 
https://github.com/ls1intum/Artemis/pull/4383

Reply:
Agree, changed.